env.default

```diff
@@ -19,7 +20,7 @@ AUTHTKT_SECRET=io7heoDui8xaikie1rushaeGeiph8Bequei6ohchaequob6viejei0xooWeuvohf
 AUTHTKT_COOKIENAME=auth_tkt_geomapfish
 AUTHTKT_HTTP_ONLY=True
 AUTHTKT_SECURE=True
-AUTHTKT_SAMESITE=Lax
+AUTHTKT_SAMESITE=None
```

Comment:
Why?

Reply:
sorry, I commited the file by mistake, I believe.
Thanks